src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3596,6 +3597,20 @@
     public static final int RECORD_DEF =
             GeneratedJavaTokenTypes.RECORD_DEF;
 
+    /**
+     * The {@code yield} keyword.  This element appears
```

Comment:
Not directly related  to this line, but what should we do in the way of updating the switch statement documentation?  Have two different examples, then include a yield in each?